src/components/common/CoinSendingModal.tsx

```diff
@@ -178,9 +177,9 @@ const CoinSendingModal: React.FC<{
         </Form.Item>
         <Form.Item label={formatMessage(messages.increaseCoins)} name="amount">
           <InputNumber
-            min={1}
-            formatter={value => (parseInt(`${value}`) >= 0 ? `+${value}` : `${value}`)}
-            parser={value => value?.replace(/\D/g, '') || ''}
+            // min={1}
+            formatter={value => (Number(`${value}`) > 0 ? `+${value}` : `${value}`)}
+            // parser={value => value?.replace(/\D/g, '') || ''}
```

Comment:
could remove unused code

Reply:
I was wondering if one day this change might be reverted though, but I will amend it as what you suggest. Thanks!